### PR TITLE
Add support for querying Kanboard tasks. Fixes issue #162

### PR DESCRIPTION
**Pull request by willtunnels**

### CODE REVIEW

**Comment by dkess** [comment]
What is `usr` supposed to mean here? I tried running the manual test with my personal API key and username and I got a 401 error

**Comment by willtunnels** [comment]
`usr` is supposed to be the username you use to login to Kanboard. It worked for me and I don't have admin privileges or anything. If you do have admin privileges it's possible you actually used the application key (which you can see as an admin) instead of your user key. Your user key should be under My profile -> Actions -> API. If you've never explicitly clicked a button to generate your key, it's the application key. The application key has to be used with usr="jsonrpc" and that's what the irc bot should do. See [the docs](https://docs.kanboard.org/en/latest/api/introduction.html). If that's not the problem idk what's happening.

**Comment by abizer** [comment]
add some docstrings if possible but otherwise this lgtm

File: ocflib/infra/kanboard.py [hunk 1]
```diff
@@ -34,6 +38,19 @@ def __str__(self):
 
     @classmethod
     def from_number(cls, usr, api_key, num):
+        """Gets information about a Kanboard task based on its number.
+
+        Example usage:
+
+            KanboardTask.from_number('jsonrpc',
+                '19ffd9709d03ce50675c3a43d1c49c1ac207f4bc45f06c5b2701fbdf8929', 1)
```
Comment (dkess) [inline ocflib/infra/kanboard.py]: Is this a real API key?

Comment (willtunnels) [reply ocflib/infra/kanboard.py]: No, it's not. I copied it from an example on the Kanboard API docs.

**Pull request merged**